recipes/cotire/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "1.7.8" %}
+{% set sha256 = "84be4ff66248ba629666bed5c2b76b869db7f04425e44bba3555bbc25d5701c1" %}
+
+package:
+  name: cotire
+  version: {{ version }}
+
+source:
+  fn: cotire-{{ version }}.tar.gz
+  url: https://github.com/sakra/cotire/archive/cotire-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Include cotire in example CMakeLists.txt.
+    #
+    # https://github.com/sakra/cotire/pull/100
+    #
+    - PR_100.diff
```

Comment:
Patch from this PR ( https://github.com/sakra/cotire/pull/100 ). Needed to build the example.